Assets/Scripts/TrainingArena.cs

```diff
@@ -143,6 +143,7 @@ public void ResetArena()
         ArenaConfiguration newConfiguration = _environmentManager.GetConfiguration(arenaID);
 
         ApplyNewArenaConfiguration(newConfiguration);
+        Debug.Log($"Setting passMark to: {newConfiguration.passMark}");
```

Comment:
See https://github.com/Kinds-of-Intelligence-CFI/animal-ai-unity/pull/40#discussion_r1740757144.

Reply:
But this debug log isn't colocated with what it's saying is happening; say in the future we refactor `ApplyNewArenaConfiguration` so that it doesn't change `passMark` any more - the developer making the change has to remember that this log is happening in a different place and that they need to update it to (or they forget and we end up with a confusing logline)